kubernetes/loculus/values.yaml

```diff
@@ -1849,10 +1854,22 @@ defaultOrganisms:
       - <<: *preprocessing
         configFile:
           <<: *preprocessingConfigFile
-          log_level: INFO
-          nextclade_dataset_name: community/hodcroftlab/enterovirus/enterovirus/linked
+          minimizer_index: "https://raw.githubusercontent.com/alejandra-gonzalezsanchez/loculus-evs/master/evs_minimizer-index.json"
```

Comment:
@corneliusroemer idea: add a flag that we are multipath and that the genes need to have the subtype as a prefix